tests/loops/test_loops.py

```diff
@@ -923,7 +923,8 @@ def __init__(self, *args, dataloader: DataLoader, **kwargs):
             self.dataloader = dataloader
 
         def _shutdown_workers(self):
-            setattr(self.dataloader, "has_shutdown_workers", True)
+            has_shutdown_workers = getattr(self.dataloader, "has_shutdown_workers", 0)
```

Comment:
Need to change the variable from `has_shutdown_workers` to `workers_shutdown`, as the type is changed from Boolean

Reply:
Done !